Server/mods/deathmatch/logic/CAPI.cpp

```diff
@@ -0,0 +1,81 @@
+#include "StdInc.h"
+#include "CAPI.h"
+#include "CGame.h"
+#include "CMainConfig.h"
+#include <SharedUtil.String.h>
+
+namespace Endpoints
+{
+    inline const std::string API_ENDPOINT = "/api";
+    namespace Server
+    {
+        inline const std::string PLAYERS = "/players";
+        inline const std::string MAX_PLAYERS = PLAYERS + "/max";
+    }
+}
+
+const bool GetNextPathFragment(std::string& path) noexcept
```

Comment:
`const bool` as a returning value?

Reply:
oops, it was a lambda function and I forgot to remove `const`